### PR TITLE
RELEASE_1

### PR DESCRIPTION
**Pull request by acsoric**

### CODE REVIEW

**Review by bbutunoiops — APPROVED** [review]
Looks good to merge

**Comment by pablohernandezops** [comment]
Looks good, let´s merge it.

**Review by pablohernandezops — APPROVED** [review]

File: helm_charts/mdcore-aws-eks-values.yml [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+
+## Uncomment the following lines to set a fixed user and password for the MD Core PostgreSQL database that will overwrite 
+## any secret that already exists for these credentials (the secret will be kept when the chart is deleted)
+## If left unset the following will apply:
+## - if no "mdcore-postgres-cred" secret exists, it will be created with the user "postgres" and a random password
+## - if the "mdcore-postgres-cred" secret exists, the values from the secret are used as credentials
+
+# db_user: postgres
+# db_password: mdcorepostgres
```
Comment (sorinsubulescuopswat) [inline helm_charts/mdcore-aws-eks-values.yml:19]: Maybe the password should be left blank or some placeholder should be used. If left like this, a default password is being suggested, giving a hint that someone may use it as it is.

Comment (acsoric) [reply helm_charts/mdcore-aws-eks-values.yml:19]: We'll change it in the next release to set it by default to "null", that should trigger the chart to generate a random one on install if left uncommented.

**Pull request merged**